Functions/FirebaseFunctions/Public/FirebaseFunctions/FIRFunctions.h

```diff
@@ -66,7 +66,17 @@ NS_SWIFT_NAME(Functions)
  * See https://firebase.google.com/docs/functions/local-emulator
  * @param origin The origin of the local emulator, such as "http://localhost:5005".
  */
-- (void)useFunctionsEmulatorOrigin:(NSString *)origin NS_SWIFT_NAME(useFunctionsEmulator(origin:));
+- (void)useFunctionsEmulatorOrigin:(NSString *)origin
+    NS_SWIFT_NAME(useFunctionsEmulator(origin:))
+        __attribute__((deprecated("Use setEmulator(host:port:) instead.")));
+
+/**
+ * Changes this instance to point to a Cloud Functions emulator running locally.
+ * See https://firebase.google.com/docs/functions/local-emulator
+ * @param host The host of the local emulator, such as "http://localhost".
```

Comment:
In all cases `host` should *not* include the `http://` or any other prefix.  We need to add that internally (and `https://` is not allowed)

Reply:
Removed.